### PR TITLE
WG - Updated which tabs appear based on user roles

### LINKED ISSUE #11: BUG: Rider can see tabs in navbar they cant/shouldnt access
### Summary

A user with ONLY a rider role should not be able to see field relevant to other roles and only ride request should be available to them, but right now they can see a dropdown for `Shifts` and `Drivers Page`

### Expected Behavior

The user should only see the dropdown for `Ride Request`

### Current / Observed Behavior

image

### Steps to Reproduce

1. Log in as user with no roles
2. Get your role toggled by an admin to be a rider
3. View navbar

Implementation Todo:

- [x] Edit `AppNavbar.js` and make sure that the tabs are not available to riders

### PR DESCRIPTION
**Pull request by walexgreen**
When the user only has "Rider" set to true, the navbar only shows "Ride Request" dropdown. 

- Only shows "Shifts" dropdown if the user is an "Admin"
- Only shows "Drivers Page" button if the user is an "Admin" or a "Driver"

image

image

Closes #11

### CODE REVIEW

**Review by Tommygithubaccount123 — DISMISSED** [review]
LGTM

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                !hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && (
+                hasRole(currentUser, "ROLE_ADMIN") && isParticipant(currentUser) && (
```
Comment (pconrad) [inline frontend/src/main/components/Nav/AppNavbar.js]: Do we need the && isParticipant here?

I think that if you are an admin, you are automatically a participant, no?

**Review by pconrad — CHANGES_REQUESTED** [review]
One question....

**Review by pconrad — CHANGES_REQUESTED** [review]
One question...

File: frontend/src/main/components/Nav/AppNavbar.js (hunk 1 above)
Comment (walexgreen) [reply frontend/src/main/components/Nav/AppNavbar.js]: Yes, it appears the && isParticipant check is redundant. I updated the code.

**Review by Tommygithubaccount123 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**